### PR TITLE
Create 2019-12-01-countdown-to-the-new-year-ember-sortable.md

### PR DESCRIPTION
**Pull request by MelSumner**
Please don't merge this until December 1, 2019 (unless we have a way to not show future blog posts?)

If merged, this PR creates a blog post for the first December blog series.

### CODE REVIEW

File: source/2019-12-01-countdown-to-the-new-year-ember-sortable.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+To kick things off, we're going to look at [ember-sortable](https://emberobserver.com/addons/ember-sortable). 
+
+### What It Does
+
+The `ember-sortable` addon provides sortable UI primitives for Ember apps- this is super useful in the sense that teams don't have to reinvent this very common pattern every time they need it for their own applications! 
+
+### Why I Like It
+
+While the ember-sortable addon has been around for some time now, providing an consistent way to add drag and drop functionality to an Ember app, it is the most recent additions that have made this addon especially worth paying attention to. [Yicheng Gong](https://github.com/ygongdev), a software engineer at LinkedIn, noticed that the addon, while popular, was missing accessibility support. The addon ended up making its way to the [Adopted Ember Addons org](https://github.com/adopted-ember-addons/), a place where addons can obtain new maintainers and support.
```
Comment (drewlee) [inline source/2019-12-01-countdown-to-the-new-year-ember-sortable.md]: "providing an consistent" should be "providing a consistent"

Comment (drewlee) [reply source/2019-12-01-countdown-to-the-new-year-ember-sortable.md]: This first sentence is a bit difficult to understand on first read. Can we simplify it or break it down into a couple of sentences?

Comment (MelSumner) [reply source/2019-12-01-countdown-to-the-new-year-ember-sortable.md]: Sure!

File: source/2019-12-01-countdown-to-the-new-year-ember-sortable.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Countdown to The New Year: Ember Sortable
+author: Melanie Sumner
+tags: 2019, Addons, DecEmber
+alias: 2019-12-01-countdown-to-the-new-year-ember-sortable.md
+responsive: true
+---
+
+<b>This is the first in our DecEmber series- <i>"Countdown to The New Year: 31 Days of Ember Addons"</i>.  We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!</b>
```
Comment (amyrlam) [inline source/2019-12-01-countdown-to-the-new-year-ember-sortable.md]:
```suggestion
**This is the first in our DecEmber series- <span style="font-style: italic;">"Countdown to The New Year: 31 Days of Ember Addons"</span>. We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
```

Unfortunately `<b>` and `<i>` don't work in the repo. This is what's been working for me

File: source/2019-12-01-countdown-to-the-new-year-ember-sortable.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+To kick things off, we're going to look at [ember-sortable](https://emberobserver.com/addons/ember-sortable). 
+
+### What It Does
+
+The `ember-sortable` addon provides sortable UI primitives for Ember apps- this is super useful in the sense that teams don't have to reinvent this very common pattern every time they need it for their own applications! 
+
+### Why I Like It
+
+While the ember-sortable addon has been around for some time now, providing a consistent way to add drag and drop functionality to an Ember app, it is the most recent additions that have made this addon especially worth paying attention to. [Yicheng Gong](https://github.com/ygongdev), a software engineer at LinkedIn, noticed that the addon was missing accessibility support. He took on the work via the [Adopted Ember Addons org](https://github.com/adopted-ember-addons/), and updated the addon to add the missing features!
```
Comment (amyrlam) [inline source/2019-12-01-countdown-to-the-new-year-ember-sortable.md:21]:
```suggestion
While the ember-sortable addon has been around for some time now, providing a consistent way to add drag and drop functionality to an Ember app, it is the most recent additions that have made this addon especially worth paying attention to. [Yicheng Gong @ygongdev](https://github.com/ygongdev), a software engineer at LinkedIn, noticed that the addon was missing accessibility support. He took on the work via the [Adopted Ember Addons org](https://github.com/adopted-ember-addons/), and updated the addon to add the missing features! 
```

We've been using this style on the Ember Times, seems helpful for folks that are known by their handle or the other way around, known by their name. Feel free to ignore though!

**Review by amyrlam — APPROVED** [review]
👍 nice, never used that addon! looks good, left some optional comments!

you can check how it will look on the webpage itself by changing the date to today ... then run it locally or see it in netlify on the PR

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for initiating this series - looking forward to it!

**Pull request merged**